app/components/forms/wizard/basic-details-step.js

```diff
@@ -182,11 +182,11 @@ export default Component.extend(FormMixin, {
   }),
 
   canAcceptPayPal: computed('data.event.paymentCurrency', function() {
-    return find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).paypal;
+    return (this.get('settings.paypalSandboxUsername') || this.get('settings.paypalLiveUsername')) && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).paypal;
```

Comment:
Since this computed property depends on these, make sure you specify that in the computed property definition so that this updates when those change